app/service/statistics.py

```diff
@@ -55,26 +54,6 @@ def create_stats_dict():
     return stats_dict
 
 
-def format_monthly_template_notification_stats(year, rows):
-    stats = {
-        datetime.strftime(date, "%Y-%m"): {}
-        for date in [datetime(year, month, 1) for month in range(4, 13)]
-        + [datetime(year + 1, month, 1) for month in range(1, 4)]
-    }
-
-    for row in rows:
-        formatted_month = row.month.strftime("%Y-%m")
-        if str(row.template_id) not in stats[formatted_month]:
-            stats[formatted_month][str(row.template_id)] = {
-                "name": row.name,
-                "type": row.template_type,
-                "counts": dict.fromkeys(NOTIFICATION_STATUS_TYPES, 0),
-            }
-        stats[formatted_month][str(row.template_id)]["counts"][row.status] += row.count
-
-    return stats
-
-
```

Comment:
@A-Shumway42 was this removal intentional?  It seems like it would still be needed so I wanted to check.

Reply:
Resolving this comment in favor of the other one!